src/components/NewClusterModal.js

```diff
@@ -985,7 +1004,15 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
         // TODO PD: What do we do with this?
         !sparkMode && !isPersistentDisk && div([
           p(['Time to upgrade your compute runtime. Terra’s new persistent disk feature will safegard your work and data.']),
-          h(Link, { onClick: () => this.setState({ viewMode: 'aboutPersistentDisk' }) }, ['Learn more'])
+          h(Link, {
+            onClick: async () => {
```

Comment:
Since this `onClick` handler is the same as the other one, we could extract it into a helper method that handles the state transition and the event logging, and call it from both places.

Reply:
good point!